tests/unit/recommenders/utils/test_gpu_utils.py

```diff
@@ -57,5 +57,6 @@ def test_tensorflow_gpu():
 
 
 @pytest.mark.gpu
+@pytest.mark.skip(reason="This function in PyTorch is flaky")
```

Comment:
Maybe the problem is due to a cuda version mismatch and can be solved by installing pytorch-cuda as [here](https://pytorch.org/get-started/previous-versions/#linux-and-windows-6)?